specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualNetworkGateway.json

```diff
@@ -1044,6 +1160,122 @@
           "final-state-via": "azure-async-operation"
         }
       },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/connections/{virtualNetworkGatewayConnectionName}/startpacketcapture": {
```

Comment:
This and other new path should be one level higher. This placement is causing CI errors
![image](https://user-images.githubusercontent.com/8080186/64554474-5580d000-d2f0-11e9-9e34-cac2b95d07a1.png)

Reply:
I have changed it. Can you please Review this again ?